modules/cnf-configuring-kubelet-nro.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-configuring-kubelet-config-nro_{context}"]
+= Creating a KubeletConfig CRD
+
+The recommended way to configure a single NUMA node policy is to apply a performance profile. Another way is by creating and applying a `KubeletConfig` custom resource (CR), as shown in the following procedure: 
+
+.Procedure 
+
+. Create the `KubeletConfig` custom resource (CR) that configures the pod admittance policy for the machine profile:
+
+.. Save the following YAML in the `nro-kubeletconfig.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: worker-tuning
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: "" <1>
+  kubeletConfig:
+    cpuManagerPolicy: "static" <2>
+    cpuManagerReconcilePeriod: "5s"
+    reservedSystemCPUs: "0,1" <3>
+    memoryManagerPolicy: "Static" <4>
+    evictionHard:
+      memory.available: "100Mi"
+    kubeReserved:
+      memory: "512Mi"
+    reservedMemory:
+      - numaNode: 0
+        limits:
+          memory: "1124Mi"
+    systemReserved:
+      memory: "512Mi"
+    topologyManagerPolicy: "single-numa-node" <5>
+----
+<1> Adjust this label to match the the `machineConfigPoolSelector` in the `NUMAResourcesOperator` CR.
+<2> Adjust this based on the CPU on your nodes.
```

Comment:
There isn't something right with the numbering here. 

    cpuManagerPolicy: "static" <2>

however, it's description is:
<2> Adjust this based on the CPU on your nodes.

Based on the callouts, it look like     cpuManagerPolicy: "static" <2> should be <3> For `cpuManagerPolicy`, `static` must use a lowercase `s`.

Reply:
good catch thanks